src/config/dynamic-theme-fixes.config

```diff
@@ -113,6 +113,15 @@ IGNORE INLINE STYLE
 
 ================================
 
+*.americanexpress.com
+
+CSS
+select.css-1ui3puo > option {
```

Comment:
change to:
`select[aria-labelledby="bankAccount-label"] > option`

this should be better than class which can be generated name and can change in the future. 😉

Reply:
Will go in and change later today.

Reply:
Sorted,